config/settings.yml

```diff
@@ -250,29 +250,6 @@ discourse_ai:
       - gpt-3.5-turbo
       - gpt-4
       - claude-2
-  ai_bot_enabled_chat_commands:
```

Comment:
We need a small migration to delete the setting values from the DB too for these two deleted settings

Reply:
oh i had it on local ... checking it in